lib/content/message/headways/paging.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Content.Message.Headways.Paging do
+  defstruct [:destination, :vehicle_type, :range]
+
+  @type vehicle_type :: :bus | :trolley | :train
+  @type t :: %__MODULE__{
+          destination: PaEss.destination() | nil,
+          vehicle_type: vehicle_type(),
+          range: Headway.HeadwayDisplay.headway_range() | nil
+        }
+
+  defimpl Content.Message do
+    @default_page_width 24
+    def to_string(%Content.Message.Headways.Paging{
+          destination: nil,
+          vehicle_type: type,
+          range: range
+        }) do
+      [
+        {"#{signify_vehicle_type(type) |> String.capitalize()} every", 3},
+        {Headway.HeadwayDisplay.format_paging_headway_range(range), 3}
+      ]
+    end
+
+    def to_string(%Content.Message.Headways.Paging{
+          destination: destination,
+          vehicle_type: vehicle_type,
+          range: range
+        }) do
+      [
+        {destination_vehicle_string(destination, vehicle_type), 3},
+        {destination_range_string(destination, range), 3}
+      ]
+    end
+
+    @spec signify_vehicle_type(Content.Message.Headways.Paging.vehicle_type()) :: String.t()
```

Comment:
Hmm good point. We actually don't use "Trolleys" even on the Mattapan line and bus signs aren't even enabled for headways, most likely because they're too variable/unreliable. Will go with hardcoding "trains" for the paging headways